src/Servers/Connections.Abstractions/src/Microsoft.AspNetCore.Connections.Abstractions.csproj

```diff
@@ -24,11 +26,4 @@
     <AdditionalFiles Include="PublicAPI/$(TargetFramework)/PublicAPI.Unshipped.txt" />
   </ItemGroup>
 
-  <!-- Special case building from source because Microsoft.Bcl.AsyncInterfaces isn't available for source builds. -->
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' OR
-      '$(TargetFramework)' == '$(DefaultNetFxTargetFramework)' OR
-      ('$(MSBuildRestoreSessionId)' == '' AND '$(DotNetBuildFromSource)' != 'true') ">
```

Comment:
Don't you need to keep some of this?

Reply:
Ping for this, otherwise PR is green

Reply:
I don't think so. There's `<TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">$(DefaultNetCoreTargetFramework)</TargetFrameworks>` above and the condition already excludes the dependency when targeting `DefaultNetCoreTargetFramework`.